src/frontend/src/lib/derived/address.derived.ts

```diff
@@ -1,13 +1,22 @@
+import { BTC_MAINNET_TOKEN_ID } from '$env/tokens.btc.env';
 import { ETHEREUM_TOKEN_ID } from '$env/tokens.env';
 import { addressStore } from '$lib/stores/address.store';
-import type { OptionEthAddress } from '$lib/types/address';
+import type { OptionBtcAddress, OptionEthAddress } from '$lib/types/address';
 import { isNullish } from '@dfinity/utils';
 import { derived, type Readable } from 'svelte/store';
 
 export const addressNotLoaded: Readable<boolean> = derived([addressStore], ([$addressStore]) =>
 	isNullish($addressStore)
 );
 
+export const btcAddressMainnet: Readable<OptionBtcAddress> = derived(
+	[addressStore],
+	([$addressStore]) =>
+		$addressStore?.[BTC_MAINNET_TOKEN_ID] === null
```

Comment:
We can maybe create a function to avoid code duplication?

Reply:
done!

Reply:
Sorry... in another PR #2066